views/editor/userbar.html

```diff
@@ -17,6 +17,9 @@
       <div id="navbar-save-indicator" class="hide">
         {{ gettext("fileSavingIndicator") }}
       </div>
+      <div id="navbar-saved-offline-indicator">
```

Comment:
you want this `div` only when the user is logged in. So this code needs to wrapped in a nunjucks `if` block similar to https://github.com/mozilla/thimble.mozilla.org/blob/master/views/editor/userbar.html#L101-L106